README.md

```diff
@@ -10,6 +10,34 @@ Contents of this repo:
 * `cmd/goacme/` - cli tool, similar to the official `letsencrypt`
 * `appengine/` - Google App Engine for Go package
 
+## Usage
+
+Optionally, if you would not like `goacme` to generate the account key for you, you should create an unencrypted private key for use with account registration. This can be done using OpenSSL:
+
+    mkdir -p ~/.config/acme
+    openssl -out ~/.config/acme/account.key 2048
+
+Next, register an account using the `goacme` client. The contact information, such as email address at the end, is optional but recommended. If you would like to have `goacme` generate the account key for you instead of using OpenSSL or similar software, include the `-gen` flag. Both example are shown below, though only one should be run depending on the whether you are providing the account key or having `goacme` generate one.
+
+    # Manually Generated Account Key
+    goacme reg mailto:your-email-address@gmail.com
+
+    # Automatically Generated Account Key
+    goacme reg -gen mailto:your-email-address@gmail.com
+
+Next, review the terms and if you agree, you can accept them like so:
+
+    goacme update -accept
+
+    Now, we are ready to request our certificate. The certificate will be placed alongside key file, specified with the `-k` argument. If the key file does not exist, a new one will be created. If you would prefer to generate the key yourself, this may be done using openssl or a similar tool, and has been included / shown in the example below. `goacme cert` currently implements only the HTTP challenge mechanism, which requires the command to be run in a way the challenge can be resolved on the same machine, i.e. it's running a local HTTP server for the duration of authorization phase.
```

Comment:
Two last (I hope) things:

The line is very long, can you split it in around 80 chars each

"... requires the command to be run in the same machine" is not strictly true anymore. I added a `-manual` flag, see this for more info: https://github.com/google/goacme/commit/d14a85d62e085a883cb7d8ba03131dcd668e60bd

Essentially, it allows you to run `goacme cert` on a different host, as long as you have access to where the domain is served from.